.golangci.yml

```
@@ -54,7 +54,9 @@ linters-settings:
           alias: utilruntime
         - pkg: k8s.io/client-go/kubernetes/scheme
           alias: clientgoscheme
-        # Controller Runtime  
+        - pkg: github.com/rancher-sandbox/rancher-turtles/util/naming
```

let's not add it under k8s pkg, but rather put them all separately under `Rancher Turtles`:
```suggestion
       # Rancher Turtles
       - pkg: github.com/rancher-sandbox/rancher-turtles/util/naming
         alias: turtlesnaming
       - pkg: github.com/rancher-sandbox/rancher-turtles/internal/rancher/provisioning/v1
         alias: provisioningv1
       - pkg: github.com/rancher-sandbox/rancher-turtles/internal/rancher/management/v3
         alias: managementv3
```